### DISCUSSION TITLE
[Help Wanted] Is there a User guide?

### DISCUSSION THREAD

**Discussion by y59** [standardnotes/forum#3466] [General]
There are many things about how to use Standard Notes that I do not understand. I dont find what I need on the Help/FAQ page. It seems my only option is to go to this Forum, where there are answers to all manner of questions, but I dont see a systematic introduction to Standard Notes. I search for key terms in the forums and scroll and scroll, but I dont find the simple information that I am looking for. 

For example, this is my experience: I just want to sign out of the app on IOS. But to do this simple act, it seems there is much I have to research before I can sign out! I go to account/ sign out workspace. It says "Sign out workspace? This action will remove this workspace and its related data from this device. Your synced data will not be affected." I dont know what this means. Is it safe to do or will I lose my data. I dont know enough to tell. It sounds like I will lose my data. That is scary. But maybe not if my data is synced. I have to research this before i take action. What is a workspace? How is it defined? I dont know. I go to the Help/ Faq page and find nothing about the consequences of signing out of a workspace, and if I do word search I dont find the word Workspace. So I go to this forum and search for Workspace and all kinds of very specific problems come up. I would just like a user guide. Or else my progress in adjusting to using this app is very slow. It seems like a great app, and I would appreciate an introduction to how to use it.

**Comment by ghost** [comment]
I second this. As a new user it's hard to find complete documentation of SN features. The introduction notes in the SN app, and the help pages on the website are fine. But beyond that, I have to check out the release notes and manually search this issue tracker and Reddit to discover useful features such as keyboard shortcuts etc. 

This is confusing. At first I got the impression that SN's features were a lot more limited than they actually are.
